src/js/components/DataFilters/__tests__/DataFilters-test.tsx

```diff
@@ -45,6 +45,18 @@ describe('DataFilters', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
+  test('layer', () => {
+    const { container } = render(
+      <Grommet>
+        <Data data={data}>
+          <DataFilters layer />
+        </Data>
+      </Grommet>,
+    );
+
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
Is this test capturing the layer? or do we have to set it to open to capture the layer in the snapshot?